specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/policyEvents.json

```diff
@@ -0,0 +1,1006 @@
+{
+  "swagger": "2.0",
```

Comment:
>swagger [](start = 3, length = 7)

Is this a completely new API or is there an older previously approved version that can be compared with this one?

Reply:
Or are you backfilling a missing swagger?

---
In reply to: [417722565](https://github.com/Azure/azure-rest-api-specs/pull/7909#discussion_r417722565) [](ancestors = 417722565)

Reply:
these 2018-07-01-preview and 2019-10-01 both are older approved versions but they did not have paging enabled, my change in PR is to add x-ms-pageable and examples for that.